ckanext/switzerland/helpers.py

```diff
@@ -533,3 +533,30 @@ def showcase_types():
 def get_showcase_type_name(showcase_type):
     type_string = showcase_types_mapping.get(showcase_type, showcase_type)
     return get_localized_value(parse_json(type_string))
+
+
+def package_has_group(group_name, groups):
+    for group in groups:
+        if group_name == group['name']:
+            return True
+    return False
+
+
+def get_group_list():
```

Comment:
Also here I would suggest to call it get_localized_group_list instead. This is a function that might get reused and naming it correctly encourages that.